drivers/backup/auth.go

```diff
@@ -270,7 +296,8 @@ func GetCommonHTTPHeaders(userName, password string) (http.Header, error) {
 // GetPxCentralAdminPwd fetches password from PxCentralAdminUser from secret
 func GetPxCentralAdminPwd() (string, error) {
 
-	secret, err := k8s.Instance().GetSecret(PxCentralAdminSecretName, GetPxBackupNamespace())
+	pxBackupNamespace, _ := GetPxBackupNamespace()
```

Comment:
Would be good if you handle the error her itself mostly expecting users to make mistake while running the job?

Reply:
Yes that would be good.